Contest_API.md

```diff
@@ -1178,16 +1178,17 @@ Returned data:
 
 ### Accounts
 
-The accounts used for access to the contest.
+The accounts used for accessing the contest, including the account representing the client.
```

Comment:
A bit of a nitpick:

This wording is strange. Of course a list of all the accounts used for accessing the contest would include the account currently used. 

Also, I think "currently used" is better than "representing the client"?

I *think* what you want to say is something like:
>  The accounts used for accessing the contest, as well as information about the account currently used for accessing the API.

?

Reply:
Changed.